conda_build/main_develop.py

```diff
@@ -32,6 +36,12 @@ def main():
         nargs='+',
         help="Path to the source directory."
     )
+    p.add_argument(
+                   '-l', '--post-link',
```

Comment:
You can test against: https://github.com/sandhujasmine/CythonExample

One reason to not do this by default is more flexibility. In this case since conda develop doesn't yet build the extensions, user can do ./setup.py develop, then invoke --post-link. It gives the flexibility to use --post-link with setup commands.

Once we add a --build_ext command to conda develop, it should by default do the post-link after building extensions.